autocodesign/autocodesign.go

```diff
@@ -184,41 +184,49 @@ func (m codesignAssetManager) EnsureCodesignAssets(appLayout AppLayout, opts Cod
 
 	}
 
-	localCodesignAssets, missingCodesignAssets, err := m.localCodeSignAssetManager.FindCodesignAssets(appLayout, distrTypes, certsByType, devPortalDeviceUDIDs, opts.MinProfileValidityDays)
-	if err != nil {
-		return nil, fmt.Errorf("failed to collect local code signing assets: %w", err)
-	}
+	codesignAssetsByDistributionType := map[DistributionType]AppCodesignAssets{}
 
-	printExistingCodesignAssets(localCodesignAssets)
+	for _, distrType := range distrTypes {
+		localCodesignAssets, missingCodesignAssets, err := m.localCodeSignAssetManager.FindCodesignAssets(appLayout, distrType, certsByType, devPortalDeviceUDIDs, opts.MinProfileValidityDays)
```

Comment:
missingCodesignAssets is an AppLayout, IMO missingAppLayout would better describe this var